platforms/chibios/drivers/spi_master.h

```diff
@@ -63,6 +63,14 @@
 #    endif
 #endif
 
+#ifndef SPI_SCK_PAL_CURRENT_LEVEL
+#    if defined(WB32F3G71xx)
```

Comment:
The whole point of `platforms/chibios/chibios_config.h` is so this doesnt happen

Reply:
This still needs resolving.